tests/foreman/cli/test_user.py

```diff
@@ -111,18 +140,13 @@ def test_positive_create_user_1(self, test_data):
         @Assert: User is created
 
         """
-        user = make_user(test_data)
-        self.__assert_exists(user)
+        include_list = [gen_string("alphanumeric", 100)]
```

Comment:
I know very well that single use variables are not encouraged in any programming language.  Here I use it to make the code more readable.  I used it here primarily because:
* This provides a clear distinction that this is an additional list of data points that need to be tested in addition to the data supplied by valid_data_user().  
* This list will easily grow or reduce depending on the future test scenarios.   In that case, we just need to append or remove from include_list and do not touch the for loop.